code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -2,48 +2,50 @@
 	name = "Guardian Spirit"
 	real_name = "Guardian Spirit"
 	desc = "A mysterious being that stands by it's charge, ever vigilant."
-	speak_emote = list("intones")
+	speak_emote = list("hisses")
 	response_help  = "passes through"
 	response_disarm = "flails at"
 	response_harm   = "punches"
-	icon = 'icons/mob/mob.dmi'
-	icon_state = "stand"
-	icon_living = "stand"
-	icon_dead = "stand"
+	icon = 'icons/mob/guardian.dmi'
+	icon_state = "magicOrange"
+	icon_living = "magicOrange"
+	icon_dead = "magicOrange"
 	speed = 0
 	a_intent = "harm"
 	stop_automated_movement = 1
 	floating = 1
 	attack_sound = 'sound/weapons/punch1.ogg'
 	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
 	minbodytemp = 0
-	maxbodytemp = 10000000
+	maxbodytemp = INFINITY
 	attacktext = "punches"
-	maxHealth = 100000 //The spirit itself is invincible
-	health = 100000
-	environment_smash = 0
+	maxHealth = INFINITY //The spirit itself is invincible
+	health = INFINITY
+	damage_coeff = list(BRUTE = 0.5, BURN = 0.5, TOX = 0.5, CLONE = 0.5, STAMINA = 0, OXY = 0.5) //how much damage from each damage type we transfer to the owner
+	environment_smash = 1
```

Comment:
why did you turn this on?i don't think guardians should be able to smash walls down